react/PushClientBanner/Readme.md

```diff
@@ -0,0 +1,28 @@
+## Push Client Banner
+Banner to advertise for Cozy desktop client.
+
+### Basic usage
+
+```
+<BannerClient
+  text="Ajoutez vos documents et photos à votre Cozy depuis votre ordinateur : ils seront sauvegardés en lieu sûr et accessibles à tout moment !"
```

Comment:
Not blocking, but the rest of the examples are in english, it would be cool to have this one in english too!